documentation/host/data.rst

```diff
@@ -120,14 +143,16 @@ Proceed to create a database as the postgres superuser (using your postgres user
    $ createuser --pwprompt -U postgres flexmeasures_test  # enter "flexmeasures_test" as password
    $ exit
 
+.. note:: In case you encouter the following "FAILS: sudo: unknown user postgres" you need to create "postgres" OS user with sudo rights first.
```

Comment:
Maybe add the command for that right away?

Reply:
I failed to properly add user in terminal from the first try but creating user via macOS preferences worked all right